tests/integration/test_core.py

```diff
@@ -9,7 +9,48 @@
 import swiftclient
 import swiftclient.exceptions
 import swiftclient.service
-from ops.model import Application
+from ops.model import ActiveStatus, Application
+
+
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_active(app: Application):
+    """
+    arrange: given charm has been built, deployed and related to a dependent application
+    act: when the status is checked
+    assert: then the workload status is active.
+    """
+    assert app.units[0].workload_status == ActiveStatus.name
+
+
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_hello_kubecon_reachable(hello_kubecon_url: str):
+    """
+    arrange: given charm is deployed and related with hello-kubecon and nginx-integrator
+    act: when the dependent application is queried via the ingress
+    assert: then the response is HTTP 200 OK.
+    """
+    response = requests.get(hello_kubecon_url)
```

Comment:
Add a timeout to `requests.get`, there's no default timeout for `requests` so it will wait indefinitely if there's no response.